data/org.containers.bluechi.conf.in

```diff
@@ -3,22 +3,22 @@
 <busconfig>
 
   <policy user="@dbus_srv_user@">
```

Comment:
Valid for the whole file: `org.containers.hirte.` should be replaced with `io.github.eclipse-bluechi.bluechi.` according to the doc and today's discussion